active/teachingtip/TeachingTipSpec.md

```diff
@@ -0,0 +1,572 @@
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            IsLightDismissEnabled="True"
+            Margin="15">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipLightDismissSample.jpg" height="600"></a>
+
+### Preferred placement
+
+Preferred placement of a teaching tip can be set using the TeachingTipPlacementMode property. If the preferred placement would not leave enough room for the teaching tip to show, another placement mode will automatically be chosen.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick"></Button>
+<muxc:TeachingTip x:Name="AutoSaveTip"
+    Title="Saving automatically"
+    Subtitle="We save your changes as you go - so you never have to."
+    PreferredPlacement="TopEdgeAlignedLeft"
+    Margin="20,50,0,0">
+</muxc:TeachingTip>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipPreferredPlacementSample.jpg" height="600"></a>
+
+### Escaping window bounds
+
+On Windows 19H1 and above, a teaching tip can escape window bounds by setting the ShouldConstrainToRootBounds property.
+
+XAML
+```XAML
+<Button Content="Tip Example" Click="OnFirstSaveButtonClick">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="AutoSaveTip"
+            Title="Saving automatically"
+            Subtitle="We save your changes as you go - so you never have to."
+            PreferredPlacement="BottomEdgeAlignedRight"
+            Margin="-80,-50,0,0"
+            ShouldConstrainToRootBounds="False">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+C#
+```C#
+public void OnFirstSaveButtonClick(object sender, RoutedEventArgs args)
+{
+    TeachingTipExample.IsOpen=true;
+}
+```
+
+<a href="url"><img src="TeachingTipOutOfWindowBoundsSample.jpg" height="600"></a>
+
+### Deferred close
+
+The close event of a teaching tip can be deffered to allow time for an action or custom animation to occur. 
+
+```XAML
+<Button Content="Tip Example">
+    <muxc:TeachingTip.Attach> 
+        <muxc:TeachingTip x:Name="EnableNewPrivacySettingsTip"
+            Title="Title string"
+            Subtitle="Body text in a minimum height tip."
+            Closing="OnTipClosing">
+        </muxc:TeachingTip>
+    </muxc:TeachingTip.Attach>
+</Button>
+```
+
+```C#
+Deferral tipDeferral;
+
+public void OnTipClosing(object sender, TeachingTipClosingEventArgs args)
+{
+    if (args.Reason == TeachingTipCloseReason.CloseButton)
+    {
+        using (tipDeferral=args.GetDeferral)
```

Comment:
This can just be
```
using(args.GetDeferral())
```
(and remove the declaration above)

Reply:
```suggestion
        using(args.GetDeferral())
```